e2e-tests/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'java'
+    id 'checkstyle'
+    alias(libs.plugins.allure)
+}
+
+ext {
+    kafkaVersion = "3.8.0"
+    aspectjVersion = "1.9.21"
+    allureVersion = "2.27.0"
```

Comment:
Same for these

Reply:
Also, this version does not seem correct. It should be from the confluent distribution

Reply:
This is e2e we'll make it more "gradle-style" later, current task is to migrate

Reply:
I am not following. Is the current e2e implementation using their own versions?

Migration shouldn't degrade the quality compared to main

I don't believe we are in a rush to migrate